facturacion_mexico/facturacion_fiscal/services/test_payment_entry_reclasificacion.py

```
@@ -312,3 +318,18 @@ def test_impuesto_cero_ignorado(self):
 		pe = self._run_cargar(tax_amount=0.0, grupos_override={})
 		reclas = [t for t in pe["taxes"] if _RECLAS_MARKER in (t.get("description") or "")]
 		self.assertEqual(len(reclas), 0)
+
+	def test_sin_mapeo_muestra_aviso_al_usuario(self):
+		"""Cuentas sin mapeo → msgprint naranja; PE se guarda sin bloquear."""
+		pe = _make_pe_mock(1160.0, grand_total=1160.0)
+		with mock.patch(f"{_MODULE}._calcular_grupos_desde_doc", return_value=({}, [self.otro])):
+			with mock.patch.object(payment_entry_reclasificacion.frappe, "msgprint") as mock_msg:
```

Ruff (0.15.12)

[warning] Use a single `with` statement with multiple contexts instead of nested `with` statements

(SIM117)